provider/health.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2016 IBM Corp. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import psutil
+import datetime
+
+MILLISECONDS_IN_SECOND = 1000
+MEGABYTE = 10 ** 6
+
+# https://pythonhosted.org/psutil/
+
+class Health:
```

Comment:
Seems like this doesn't need to be a class since it doesn't hold any state. Could this just be a collection of functions instead? Or maybe just one `generateReport()` function.